test-apps/display-performance-test-app/src/backend/WebMain.ts

```diff
@@ -99,13 +99,25 @@ function startWebServer() {
   // ---------------------------------------------
   // Start the browser, if given a specific one
   // ---------------------------------------------
-  if (browser === "chrome")
-    chromeLauncher.launch({ // eslint-disable-line @typescript-eslint/no-floating-promises
-      startingUrl: "http://localhost:3000",
-      chromeFlags,
-    }).then((val) => { DisplayPerfRpcInterface.chrome = val; });
-  else if (browser === "firefox")
-    child_process.execSync("start firefox http://localhost:3000");
-  else if (browser === "edge")
-    child_process.execSync("start microsoft-edge:http://localhost:3000");
+  switch (browser) {
+    case "chrome":
```

Comment:
Would it be possible to we also add the Chrome case for Macs?  Similar to what you did for Firefox below?

Reply:
Or does chrome-launcher work both places already, we think?

Reply:
Since we're using the chrome-launcher package to open and close chrome, I think it should work the same for Mac & Windows already. (though I'm hoping you or someone else can double check that for me, running on Mac once these changes are in)

Reply:
Can we test on mac before merging this PR?

Reply:
@pmconne @hnn0003 Agree - I'll try to give this a try this afternoon.

Reply:
Well, I can't, since I don't have a mac (although, I could go grab the one Dad has right now, if that's easiest), but I certainly don't think it'd be a big deal to wait to merge this in until someone can do that.

Reply:
@hnn0003 I'll let you know if I don't get time to try this branch this afternoon, or run into problems.

Reply:
@pmconne @hnn0003 We tried this on my Mac Mini and it all works now after a few fixes / changes.  Can run tests for Safari / Chrome / Firefox on a Mac.